virtcontainers/qemu.go

```diff
@@ -582,6 +584,27 @@ func (q *qemu) startSandbox(timeout int) error {
 		}
 	}()
 
+	if q.config.SharedFS == config.VirtioFS {
+		var cmd *exec.Cmd
+		sockPath, err := utils.BuildSocketPath(vmPath, "vhost-fs.sock")
+		if err != nil {
+			return err
+		}
+
+		// The daemon will terminate when the vhost-user socket
+		// connection with QEMU closes.  Therefore we do not keep track
+		// of this child process.
+		sourcePath := filepath.Join(kataHostSharedDir, q.id)
+		cmd = exec.Command(q.config.VirtioFSDaemon,
```

Comment:
I think the daemon should start after launching qemu, the fs socket is created until QEMU is started, right? what happen if QEMU can't start? the daemon will be running forever waiting for the socket?

Reply:
Will fix in the next revision.